rollup.config.js

```diff
@@ -1 +1,13 @@
-module.exports = require('skatejs-build/rollup.config');
+const config = require('skatejs-build/rollup.config');
+const pkg = require('skatejs-build/package');
```

Comment:
Shouldn't this be including the `package.json` from the current working directory, not the build?